lesson_02/main.py

```diff
@@ -0,0 +1,51 @@
+from dotenv import load_dotenv
+import os
+import shutil
+from flask import Flask, request, jsonify
+import requests
+
+load_dotenv()
+
+AUTH_TOKEN = os.getenv('AUTH_TOKEN')
+
+app = Flask(__name__)
+
+# sekret token
+AUTH_TOKEN = os.getenv('AUTH_TOKEN')
+
+# 2
+
+# 3 joba's idempotence
+import shutil
```

Comment:
best practice is to keep all import libraries in one place, at the beginning of the file